src/background/utils/db.js

```diff
@@ -208,20 +210,20 @@ export function getScriptById(id) {
 }
 
 /** @return {?VMScript} */
-export function getScript({ id, uri, meta }) {
+export function getScript({ id, uri, meta, removed }) {
   let script;
   if (id) {
     script = getScriptById(id);
   } else {
     if (!uri) uri = getNameURI({ meta, id: '@@should-have-name' });
-    script = store.scripts.find(({ props }) => uri === props.uri);
+    script = store[removed ? 'removedScripts' : 'scripts'].find(({ props }) => uri === props.uri);
   }
   return script;
 }
 
 /** @return {VMScript[]} */
 export function getScripts() {
-  return store.scripts.filter(script => !script.config.removed);
+  return store.scripts;
```

Comment:
Previously it returned a copy, so I wonder if we should use `[...store.scripts]` here (seems wasteful) or maybe `[...getScripts()]` in plugin/index.js?